tests/_internal/mbed_tools/test_export.py

```diff
@@ -13,20 +14,24 @@
 
 
 class TestExport(TestCase):
-    def test_export(self):
+    @mock.patch("mbed_build._internal.mbed_tools.export.generate_cmakelists_file")
+    def test_export(self, mock_generate_cmakelists_file):
         # This is needed to have access to the templates directory in Patcher filesystem.
         self.setUpPyfakefs()
         self.fs.add_real_directory(os.path.dirname(templates.__file__))
```

Comment:
Looking at this with fresh set of eyes. Wouldn't it be better to mock out file writing as well? This `pyfakefs` call carries a lot of _implementation detail_ knowledge. Looking at you `os.path.dirname(templates.__file__)`.

Reply:
Yeah you are right 
5a39451